alerting/provider/zulip/zulip.go

```diff
@@ -0,0 +1,131 @@
+package zulip
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/TwiN/gatus/v5/alerting/alert"
+	"github.com/TwiN/gatus/v5/client"
+	"github.com/TwiN/gatus/v5/config/endpoint"
+	"io"
+	"net/http"
+	"net/url"
+)
+
+type Config struct {
+	// BotEmail is the email of the bot user
+	BotEmail string `yaml:"bot-email"`
+	// BotAPIKey is the API key of the bot user
+	BotAPIKey string `yaml:"bot-api-key"`
+	// Domain is the domain of the Zulip server
+	Domain string `yaml:"domain"`
+	// ChannelId is the ID of the channel to send the message to
+	ChannelId string `yaml:"channel-id"`
```

Comment:
Can you rename `ChannelID` to `ChannelId`?

Reply:
You mean `ChannelId` to ChannelID` right?

Reply:
@TwiN I assumed `ChannelId` to `ChannelID` and pushed it, let me know if there is anything else